.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Hmm what does this file do? This is probably irrelevant to the portfolio, if this is the case, you can add it to .gitignore.